README.md

```diff
@@ -1 +1,137 @@
-# FIXME
+<div align="center">
+<img src="https://pl-public-data.s3.amazonaws.com/assets_lightning/Lit_LLaMA_Badge3x.png" alt="Lit-LLaMA" width="128"/>
+
+# ⚡ Lit-StableLM
+
+<!--
+<p align="center">
+  <a href="https://www.lightning.ai/">Lightning.ai</a> •
+  <a href="https://lightning.ai/docs/pytorch/stable/">PyTorch Lightning</a> •
+  <a href="https://lightning.ai/docs/fabric/stable/">Fabric</a>
+</p>
+-->
+
+![cpu-tests](https://github.com/lightning-AI/lit-stablelm/actions/workflows/cpu-tests.yml/badge.svg) <!-- [![Build Status](https://dev.azure.com/Lightning-AI/lit%20Models/_apis/build/status%2FLightning-AI.lit-StableLM?branchName=main)](https://dev.azure.com/Lightning-AI/lit%20Models/_build/latest?definitionId=49&branchName=main) --> [![license](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://github.com/Lightning-AI/lit-stablelm/blob/master/LICENSE) [![Discord](https://img.shields.io/discord/1077906959069626439?style=plastic)](https://discord.gg/VptPCZkGNa)
+
+<!-- <img src="https://pl-public-data.s3.amazonaws.com/assets_lightning/Llama_pineapple.gif" alt="Lit-LLaMA and pineapple pizza" width="500px"/> -->
+
+</div>
+
+# ⚡ Lit-StableLM
+
+Hackable implementation of the [StableLM](https://github.com/Stability-AI/StableLM) and [Pythia](https://github.com/EleutherAI/pythia) family of models released under the **Apache 2.0 license.**
+
+This implementation builds on [Lit-LLaMA](https://github.com/lightning-AI/lit-llama) and [nanoGPT](https://github.com/karpathy/nanoGPT), and it's powered by [Lightning Fabric](https://lightning.ai/docs/fabric/stable/) ⚡.
+
+Weights are available under the Apache 2.0 license and can be downloaded [following these instructions](howto/download_weights.md).
+
+## Design principles
+
+This repository follows the **Lit design principle** of **openness through clarity**.
+
+**Lit-StableLM** is:
+
+- **Simple:** Single-file implementation without boilerplate.
+- **Correct:** Numerically equivalent to the original model.
+- **Optimized:** Runs on consumer hardware or at scale.
+- **Open-source:** No strings attached.
+
+Avoiding code duplication is **not** a goal. **Readability** and **hackability** are.
+
+## Get involved!
+[Join our Discord](https://discord.gg/VptPCZkGNa) to build high-performance, truly open-source models for the common benefit of the community.
+
+&nbsp;
+
+## Setup
+
+Clone the repo
+
+```bash
+git clone https://github.com/Lightning-AI/lit-stablelm
+cd lit-llama
+```
+
+install dependencies
+
+```bash
+pip install -r requirements.txt
+```
+
+You are all set! 🎉
+
+&nbsp;
+
+## Use the model
+
+To generate text predictions, you need to download the model weights. **If you don't have them, check out our [guide](howto/download_weights.md).**
+
+Run inference:
+
+```bash
+python generate.py --prompt "Hello, my name is"
+```
+
+This will run the 7B model and require ~26 GB of GPU memory (A100 GPU).
```

Comment:
It's currently configured to run the 3b pre-trained model. We need to update these numbers. Can be done in a follow-up